app/models/allergen.rb

```diff
@@ -0,0 +1,16 @@
+# == Schema Information
+#
+# Table name: allergens
+#
+#  id         :bigint           not null, primary key
+#  name       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Allergen < ApplicationRecord
```

Comment:
We'll need to validate the name presence for this after.

Reply:
added validation (presence and uniqueness) today, and also added validations on other models